cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentialProfile.tsx

```diff
@@ -57,11 +57,17 @@ function ManagementCredentialProfile(props: Props) {
             </p>
             <p cds-layout="m-y:lg" className="description">
                 Don&apos;t have an AWS Credential profile? Credential profiles can be configured using the{' '}
-                <a href="/" className="text-blue">
+                <a
+                    href="/Users/miclettej/Dev/miclettej-community-edition/community-edition/cli/cmd/plugin/ui/web/tanzu-ui/public"
```

Comment:
We probably don't want `miclettej` in the href... (and below)

Reply:
oh what the heck. That must have happened when I did some automated refactoring through the IDE